plugins/apps/threatbus_cif3/plugin.py

```diff
@@ -0,0 +1,98 @@
+from queue import Queue
+import threading
+from cifsdk.client.http import HTTP as Client
+
+from threatbus_cif3.message_mapping import map_to_cif
+import threatbus
+
+"""Threatbus - Open Source Threat Intelligence Platform - plugin for CIFv3"""
+
+plugin_name = "cif3"
+cif = None
+lock = threading.Lock()
+
+
+def validate_config(config):
+    assert config, "config must not be None"
+    config["tags"].get(list)
+    config["tlp"].get(str)
+    config["confidence"].as_number()
+    config["group"].get(str)
+    if config["api"].get(dict):
+        config["api"]["host"].get(str)
+        config["api"]["ssl"].get(bool)
+        config["api"]["token"].get(str)
+
+
+def receive_intel_from_backbone(watched_queue, config):
+    """Reports / publishes intel items back to the given CIF endpoint.
+        @param watched_queue The py queue from which to read messages to submit on to CIF
+    """
+    global logger, cif, lock
+    logger.debug(
+        f"Entering receive_intel_from_backbone func to monitor queue {watched_queue}..."
+    )
+    if not cif:
+        logger.debug("No global CIF found. Exiting function")
+        return
+    while True:
+        intel = watched_queue.get()
+        if not intel:
+            logger.debug("Sighting msg was empty")
+            continue
+        cif_mapped_intel = map_to_cif(intel, logger, config)
+        if not cif_mapped_intel:
+            return
+        logger.info(f"CIF queue report sighting for intel {intel}")
```

Comment:
```suggestion
        logger.debug(f"Adding intel to CIF: {cif_mapped_intel}")
```

This should be a debug statement. Also, we are adding intel and not a sighting. We should print the mapped intel for debug purposes.